ehealthid-rp/src/main/resources/i18n_de.properties

```diff
@@ -0,0 +1,5 @@
+title=Anmeldung mit GesundheitsID
```

Comment:
Not an expert in German. Is `mit GesundheitsID anmelden` more idiomatic?

Reply:
This is what DeepL got me, I guess @thomasrichner-oviva  can advise on proper German :D